src/standalone/scripts/store-past-claimed-green-beans.ts

```diff
@@ -0,0 +1,98 @@
+import { NestFactory } from '@nestjs/core';
+import { decodeFunctionData } from 'viem';
+import * as dayjs from 'dayjs';
+import { AppModule } from '../../app.module';
+import { GREEN_BEAN_ADDRESS } from '../../constants';
+import { client } from '../../utils/client';
+import { greenBeanAbi } from '../../abis/green-bean-abi';
+import { GreenBeanService } from '../../services/green-bean.service';
+
+// store the thumbnail images for all Azukis
+async function bootstrap() {
+  const app = await NestFactory.createApplicationContext(AppModule);
+  const greenBeanService = app.get(GreenBeanService);
+  const logs = await client.getLogs({
+    address: GREEN_BEAN_ADDRESS,
+    event: {
+      anonymous: false,
+      inputs: [
+        {
+          indexed: true,
+          internalType: 'address',
+          name: 'operator',
+          type: 'address',
+        },
+        {
+          indexed: true,
+          internalType: 'address',
+          name: 'from',
+          type: 'address',
+        },
+        {
+          indexed: true,
+          internalType: 'address',
+          name: 'to',
+          type: 'address',
+        },
+        {
+          indexed: false,
+          internalType: 'uint256',
+          name: 'id',
+          type: 'uint256',
+        },
+        {
+          indexed: false,
+          internalType: 'uint256',
+          name: 'value',
+          type: 'uint256',
+        },
+      ],
+      name: 'TransferSingle',
+      type: 'event',
+    },
+    args: {
+      from: '0x0000000000000000000000000000000000000000',
+    },
+    fromBlock: BigInt(17601933),
+    toBlock: BigInt(17653586),
+  });
```

Comment:
perhaps this logic can be extracted into a separate function and file `getLogs()`?